.github/workflows/build-test-pr.yml

```diff
@@ -204,11 +204,12 @@ jobs:
             make pytest_fast
           fi
 
-      - name: Upload coverage artifacts
-        uses: ./.github/actions/upload_coverage_artifacts
+      - name: Upload coverage report to Codecov
         if: ${{ matrix.packageName == '*.whl' }}
-        with:
-          folder: ${{ matrix.folder }}
+        run: |
+          curl -Os https://uploader.codecov.io/latest/linux/codecov
+          chmod +x codecov
+          ./codecov
```

Comment:
Can't it be pip-installed?

Reply:
that's the suggested use in the codecov documentation, just following that.

Reply:
Shouldn't we use the GHA instead?

https://docs.codecov.com/docs/codecov-uploader#introduction

Reply:
Can we just double-check whether the GHA is the recommended setup?

Reply:
There's no clear recommendation, but I've switched to using the Github action. A lot nicer now, in my opinion.

Reply:
I agree!